.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "plato"]
```

Comment:
Use a python package as a requirement instead of submodule.

Reply:
Removed.
Due to the large package issue, we currently add Plato as third-party lib.